Support bind schemas and OSB fields in bundles (#2144)
kyma-project/kyma: docs/service-brokers/docs/012-configuration-helm-broker-bundles.md

|   **documentationURL**  |   false  |                                  The **documentationURL** field described in the [Open Service Broker API][service-metadata].                                    |
|      **supportURL**     |   false  |                                     The **supportURL** field described in the [Open Service Broker API][service-metadata].                                       |
|       **imageURL**      |   false  |     The **imageURL** field described in the [Open Service Broker API][service-metadata]. You must provide the image as an SVG.          |

> | **bindingRetrievable**  |   false  |     The **bindingRetrievable** field specifies whether fetching a ServiceBinding using a `GET` call on the resource's endpoint is supported for all plans. The default value is `false`.         |

Reviewer: please point me to docs that default value is false.

Author: I can't find a doc with this. But if we don't specify `bindingRetrievable` in `meta.yaml`, the default value for it is false in the ServiceClass.

Reviewer 2: Service-catalog does not contain any logic with the default value, so it is `false`

> |   **planUpdatable**     |   false  |     The **planUpdatable** field PlanUpdatable represents whether instances of this service may be updated to a different plan. The default value is `false`         |

Reviewer 3: ```suggestion
|   **planUpdatable**     |   false  |     The **planUpdatable** field specifies whether instances of this service can be updated to a different plan. The default value is `false`         |
```

> |       **requires**      |   false  |     The **requires** field defines a list of permissions the user must grant to the instances of this service.          |

Reviewer: Describe bind-instance-schema and update-instance-schema. I see that also "labels" are not documented, whats about metadata, labels are a subset of metadata.

Author: Described bind-instance-schema and update-instance-schema. Added description of labels field. Metadata discussed.

> |       **requires**      |   false  |     The **requires** field defines a list of permissions the user must give instances of this service.          |

Reviewer 3: ```suggestion
|       **requires**      |   false  |     The **requires** field defines a list of permissions the user must grant to the instances of this service.          |
```

| **description** |   true   | The plan description. It has the same restrictions as defined in the [Open Service Broker API][plan-objects]. |
| **displayName** |   true   | The plan display name. It has the same restrictions as defined in the [Open Service Broker API][plan-objects]. |
|  **bindable**   |   false  | The plan bindable attribute. It has the same restrictions as defined in the [Open Service Broker API][plan-objects].    |

> |     **free**    |   false  | The plan free attribute.  The default value is `false`    |

Reviewer 3: ```suggestion
|     **free**    |   false  | The attribute which specifies whether the instance is free or not. The default value is `false`.    |
```

This file is not required.
For more information about the content of the `create-instance-schema.json` file, see the [Input parameters](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#input-parameters-schema-object) specification.

#### The bind-instance-schema.json file

The `bind-instance-schema.json` file contains a schema used to define the parameters. Each input parameter is expressed as a property within a JSON object.
This file is not required.
For more information about the content of the `bind-instance-schema.json` file, see the [Input parameters](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#input-parameters-schema-object) specification.

#### The update-instance-schema.json file

> The `update-instance-schema.json` file contains a schema used to define the parameters. Each input parameter is expressed as a property within a JSON object.

Reviewer 2: Ther eis no any comment it is about an update (or the binding in the previous) operation

Author: I developed each sentence to mention an operation to which it applies.
